ampd/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM rust:latest as build
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y clang protobuf-compiler
+COPY . .
+RUN cargo build --release
+
+CMD ["./target/release/ampd"]
```

Comment:
if this is supposed to be just the build stage then there is no need to set the `CMD`